src/Model/Item.php

```diff
@@ -1328,10 +1328,11 @@ public static function insert(array $item, int $notify = 0, bool $post_local = t
 			}
 		}
 
+		if (!empty($source) && ($transmit || DI::pConfig()->get($uid, 'debug', 'store_source'))) {
```

Comment:
Who can set this? It seems like a global config, so I'd rather say:

```suggestion
		if (!empty($source) && ($transmit || DI::config()->get('debug', 'store_source'))) {
```
Or am I wrong?

Reply:
You're right, and I have to take over this branch as well to fix the `Item\Source` module, this is only half the battle.

Reply:
Yeah, that pconfig thingy was a copy&paste issue.